draft-kuehlewind-shmoo-remote-fee.md

```diff
@@ -130,7 +130,12 @@ participants, nor to any specific other restrictions like the number of meetings
 The purpose is simply to maximise participation without barriers in order to make the standards process as open as possible.
 
 It is expected that participants who have financial support to use the regular registration option 
-will do so. Aggregated data on the number and percentage of free registrations used should be published,
+will do. Paying a registration fee is way for their sponsor to support the sustainability of the IETF. Currently, e.g. even a higher late payment
```

Comment:
I would rewrite as "For example, a higher late payment charge can be used to ..."